### DISCUSSION TITLE
import saved cashflows into one another.

### DISCUSSION THREAD

**Discussion by trv893** [FlowiseAI/Flowise#61] [Ideas]
If I have two chat flows that accomplish different things, it would be nifty to be able to easily add one into the end of another.

**Comment by HenryHengZJ** [comment]
Chatflow you mean 😂
And you're right, its definitely a good idea, will add it to feature request

**Reply by trv893** [reply]
sweet! another small thing is that it would be nice to have an icon on the top level node types (agents, chains, chat models) that is also displayed when you drop one into the chatflow. 

So when looking through templates, you know if the element in the flow you are looking at is in the agent dropdown, chains dropdown, etc. 

once I get a little more familiar with all this it would obviously become less necessary, but as I am just jumping into templates and experimenting it would help keep me oriented. 

this project is amazing! thanks you!!
